tests/test_config/test_config.py

```diff
@@ -574,7 +573,7 @@ def test_nitpick_base(app, status, warning):
 
     warning = warning.getvalue().strip().split('\n')
     assert len(warning) == len(nitpick_warnings)
-    for actual, expected in zip(warning, nitpick_warnings):
+    for actual, expected in zip(warning, nitpick_warnings, strict=False):
```

Comment:
```suggestion
    for actual, expected in zip(warning, nitpick_warnings, strict=True):
```

and remove the above assert